### PR TITLE
Added PHP 7.1 functionality

### LINKED ISSUE #70: Utilize PHP 7.1 features

### PR DESCRIPTION
**Pull request by szymach**
Resolves #70

### CODE REVIEW

File: Behat/Context/DataContext.php [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
 use FSi\FixturesBundle\Entity\Comment;
+use FSi\FixturesBundle\Entity\CommentTranslation;
 use FSi\FixturesBundle\Entity\Event;
+use FSi\FixturesBundle\Entity\EventTranslation;
 use FSi\FixturesBundle\Entity\File;
 use SensioLabs\Behat\PageObjectExtension\PageObject\Exception\ElementNotFoundException;
+use SplFileInfo;
 use Symfony\Component\HttpKernel\KernelInterface;
 use Symfony\Component\PropertyAccess\PropertyAccess;
+use function file_exists;
```
Comment (rn0) [inline Behat/Context/DataContext.php]: it seems to be unused

Comment (szymach) [reply Behat/Context/DataContext.php]: https://github.com/szymach/admin-translatable-bundle/blob/c15a78068b9a4b7e9582bc7e8ced395dbcc3d332/Behat/Context/DataContext.php#L57

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Pull request merged**